src/github.com/matrix-org/dendrite/mediaapi/writers/utils.go

```diff
@@ -0,0 +1,103 @@
+// Copyright 2017 Vector Creations Ltd
```

Comment:
It happened to be, yes. I was expecting more functions to go here due to both upload and download needing to potentially write files to disk. Download proved to be different enough to need its own specific code. I shall rename the file.